.env

```diff
@@ -0,0 +1,3 @@
+DATABASE_URL=postgres://tolulopeodueke@localhost:5432/mazus_ah_backend
+TEST_DATABASE_URL=postgres://tolulopeodueke@localhost:5432/mazus_ah_backend_test
+SECRET_KEY=master
```

Comment:
What is `.env` doing on Github again???

Reply:
I didn't know this was added again, I'll remove it. Thanks

Reply:
Done